contracts/strategies/pair/PairBasedStrategyLogicLib.sol

```diff
@@ -403,7 +402,8 @@ library PairBasedStrategyLogicLib {
   function needStrategyRebalance(
     PairBasedStrategyLogicLib.PairState storage pairState,
     ITetuConverter converter_,
-    int24 tick
+    int24 tick,
+    uint poolPrice
```

Comment:
PairBasedStrategyLogicLibTest should be fixed after such change

Reply:
I see you fixed it. Thank you.